UltimateMedals.as

```diff
@@ -72,7 +75,39 @@ string fontFace = "";
 [Setting category="Display Settings" name="Font size" min=8 max=48]
 int fontSize = 16;
 
+/* Custom names */
+#if TMNEXT||MP4
+[Setting category="Display Text" name="Author Text"]
+bool authorText = true;

```

Comment:
`true` doesn't seem right

Reply:
Indeed, changing this to `string authorText = "Author"` should be fine. I must've slept over this one, as I never came to test it outside of Turbo.